doc/ref_cert/RC2/chapters/chapter02.md

```diff
@@ -311,6 +311,32 @@ The following software versions are considered to benchmark Kubernetes v1.21
 | Functest                | v1.21       |
 | xrally-kubernetes       | 1.1.1.dev12 |
 
+### Dataplane benchmarking
+
+[Kubernetes perf-tests](https://github.com/kubernetes/perf-tests) stores
+various Kubernetes-related performance test related tools especially
```

Comment:
is it tools or tools data?

Reply:
Kubernetes perf-tests hosts many benchmark tools. One of them is netperf

Reply:
@collivier Can you change "stores" in line 316 to "hosts"?